pyproject.toml

```diff
@@ -0,0 +1,86 @@
+[build-system]
+requires = ["setuptools>=61.2.0", "wheel", "setuptools_scm[toml]>=3.4.3"]
```

Comment:
This would require to drop Python 3.5 and 3.6. cf. https://github.com/pypa/setuptools/blob/v61.2.0/setup.cfg#L27
As of now, according to pypistats, there is still around ~700k, we shall wait until `requests`, `httpx`, `aiohttp`, etc.. officially drops it. (soon I think)
Dropping py 3.5 is OK today. (and is welcomed)